http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieRuntimeMonitor.scala

```diff
@@ -260,14 +260,16 @@ class ZombieRuntimeMonitor[F[_]: Parallel: ContextShift: Timer](
         s"${traceId.asString} | Deleting inactive zombie runtime: ${zombie.googleProject.value} / ${zombie.runtimeName.asString}"
       )
       _ <- metrics.incrementCounter("numOfInactiveZombieRuntimes")
-      _ <- zombie.cloudService.interpreter
-        .deleteRuntime(
-          DeleteRuntimeParams(
-            zombie.googleProject,
-            zombie.runtimeName,
-            zombie.asyncRuntimeFields.isDefined
-          ) //TODO: think about this a bit more. We may want to delete disks in certain cases
+      runtimeOpt <- clusterQuery.getClusterById(zombie.id).transaction
```

Comment:
Had to run an additional query in the zombie monitor since `deleteRuntime` now takes the entire `Runtime` object. Zombie monitor will be going away soon-ish anyway